libs/core/langchain_core/utils/image.py

```diff
@@ -0,0 +1,14 @@
+import base64
+import mimetypes
+
+
+def encode_image(image_path: str) -> str:
+    """Get base64 string from image URI."""
+    with open(image_path, "rb") as image_file:
+        return base64.b64encode(image_file.read()).decode("utf-8")
+
+
+def image_to_data_url(image_path: str) -> str:
```

Comment:
We could also make these accessible as an instance of `ImagePromptValue.to_b64_string` etc.